src/api/github/org.ts

```diff
@@ -23,22 +34,23 @@ export class GitHubOrg {
 
     // Call bindAPI ASAP. We can't call it here because constructors can't be
     // async. Note that in testing this ends up being mocked as if it were
-    // bound, even though (afaict) we generally don't call bindAPI in test.
+    // bound to an org installation, even though (afaict) we generally don't
+    // call bindAPI in test.
     this.api = new OctokitWithRetries({
       authStrategy: createAppAuth,
-      auth: this.appAuth, // unbound, good enough for now
+      auth: this.appAuth,
     });
   }
 
   async bindAPI() {
-    // Use an Octokit not bound to an org to make an Octokit bound to our org.
-    // The docs say it's safe for Octokit instances to be long-lived:
-    //
-    // > Additionally, the SDK will take care of regenerating an installation
-    // > access token for you so you don't need to worry about the one hour
-    // > expiration.
-    //
-    // https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/authenticating-as-a-github-app-installation#using-the-octokitjs-sdk-to-authenticate-as-an-app-installation
+    if (FORCE_USER_TOKEN_GITHUB_CLIENT) {
```

Comment:
For my understanding, is this easier for dev because then we can just use user PAT instead of setting up test org?

Reply:
Yes, see #473.

Reply:
I was going to add a comment but there's already a comment. :)

Reply:
Expanded the comment in 836c0050c20a4d9d9a3d800799df4697aa068ef8.